index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <script
+      src="https://kit.fontawesome.com/71d90399a2.js"
+      crossorigin="anonymous"
+    ></script>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@900&display=swap"
+      rel="stylesheet"
+    />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <section>
+      <nav>
+        <ul>
+          <li class="navName">Portfolio</li>
+          <li class="navIcon"><i class="fas fa-bars"></i></li>
+        </ul>
+      </nav>
+    </section>
+    <main>
+      <section class="nameWork">
+        <div class="heads">
+          <h1 class="nameHead">Timo Wester</h1>
+          <h3 class="jobHead">software developer</h3>
+          <div class="sideGradientleft"></div>
+        </div>
+      </section>
+      <section class="mainTextSection">
+        <div>
+          <p class="mainText">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+          <hr />
+        </div>
+      </section>
+      <section class="lists">
+        <div class="lang">
+          <ul>
+            <li class="listhead">Language</li>
+            <li>Javascript</li>
+            <li>Ruby</li>
+            <li>Html</li>
+            <li>Css</li>
+          </ul>
+        </div>
+        <div class="frameworks">
+          <ul>
+            <li class="listhead">Frameworks</li>
+            <li>Bootstrap</li>
+            <li>Ruby on Rails</li>
+            <li>RSPec</li>
+            <li>CapyBara</li>
+            <li>Selenium</li>
+          </ul>
+        </div>
+        <div class="skill">
+          <ul class="skills">
+            <li class="listhead">Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+        </div>
+      </section>
+      <hr />
+      <section>
+        <div class="sideGradientright"></div>
+        <ul class="socials">
+          <li><i class="fab fa-github"></i></li>
+          <li><i class="fab fa-linkedin"></i></li>
+          <li><i class="fas fa-hand-peace"></i></li>
+          <li><i class="fab fa-twitter"></i></li>
+          <li><i class="fab fa-facebook-square"></i></li>
+        </ul>
+      </section>
+    </main>
```

Comment:
- [ ] The button to go to the "work" section is missing, please add a `button`/`a`.

Figma design | Your design

![image](https://user-images.githubusercontent.com/60164239/133578384-bf8827be-09b3-4722-9da0-bcdbceb2f8a6.png)